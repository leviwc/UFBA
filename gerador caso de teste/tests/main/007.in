21 106
21 15
20 6
7 9
18 14
13 16
19 15
14 8
4 11
21 5
21 16
1 17
7 16
8 2
12 9
8 1
12 5
16 18
13 11
1 9
14 20
13 18
13 2
9 3
7 11
20 19
3 18
18 20
6 1
19 4
14 11
3 2
3 5
11 10
17 3
16 9
1 11
8 21
13 9
14 17
3 16
18 6
16 8
21 9
7 3
17 4
6 15
3 1
17 21
2 12
19 10
2 21
13 15
11 21
19 7
15 14
7 12
6 11
10 12
2 7
14 21
19 14
1 20
6 2
8 9
15 11
5 8
10 6
3 11
13 13
5 15
18 18
15 4
1 19
6 3
13 8
8 18
12 21
12 1
17 20
14 13
13 1
14 5
19 19
13 21
15 17
12 20
3 21
4 7
1 18
21 21
13 12
6 4
17 11
13 7
1 7
4 21
7 5
10 18
15 16
4 13
7 6
2 16
8 20
14 1
12 16
1 2
