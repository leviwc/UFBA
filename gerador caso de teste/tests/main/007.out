Resultado com busca binaria: 6
30.593000 s
Resultado com backtracking apenas: 6
0.074000 s
Resultado testando todas as possibilidades: 6
60.977000 s
Resultado com busca binaria otimizada: 6
11.866000 s
